discord/ext/commands/cog.py

```diff
@@ -165,8 +184,12 @@ class Cog(metaclass=CogMeta):
     When inheriting from this class, the options shown in :class:`CogMeta`
     are equally valid here.
     """
+    __cog_name__: ClassVar[str]
+    __cog_settings__: ClassVar[Dict[str, Any]]
+    __cog_commands__: ClassVar[List[Command]]
+    __cog_listeners__: ClassVar[List[Tuple[str, str]]]
```

Comment:
I'm not sure if I can get the vars defined in the metaclass to carrry down to the class without doing this.
Ideas?

Reply:
I've tried a few things and I can't get anything to work. This may be a bug in the metaclass type spec or in all of the type checkers.